src/NuGet.Indexing/DownloadsByVersion.cs

```diff
@@ -48,8 +41,13 @@ public int Total
 
             set
             {
+                if (_downloadsByVersion.ContainsKey(version))
+                {
+                    _total = _total - _downloadsByVersion[version];
```

Comment:
Please make sure there is a GitHub issue for the consistency issue @scottbommarito is talking about before merging, so things don't get lost, if you're not going to fix it as part of this PR.